backend/python/app/models/caregivers.py

```diff
@@ -0,0 +1,51 @@
+from sqlalchemy import inspect
+from sqlalchemy.orm.properties import ColumnProperty
+
+from . import db
+
+type_enum = db.Enum("CAREGIVER", "PROVIDER", name="caregivers_type")
+relationship_to_child_enum = db.Enum(
+    "MOM",
+    "DAD",
+    "FOSTER_MOM",
+    "FOSTER_DAD",
+    "GRANDPARENT",
+    name="caregivers_relationship_to_child",
+)
+
+
+class Caregivers(db.Model):
+    __tablename__ = "caregivers"
+    # TODO: add foreign key reference to intake
```

Comment:
made a mistake in my previous comment:
```suggestion
    # TODO: add foreign key reference to child
```